soc/arm/nuvoton_npcx/common/ecst/ecst_args.py

```diff
@@ -49,7 +49,8 @@
               'npcx7m6': {'ram_address': 0x10090000, 'ram_size': 0x40000},
               'npcx7m7': {'ram_address': 0x10070000, 'ram_size': 0x60000},
               'npcx9m3': {'ram_address': 0x10080000, 'ram_size': 0x50000},
-              'npcx9m6': {'ram_address': 0x10090000, 'ram_size': 0x40000}}
+              'npcx9m6': {'ram_address': 0x10090000, 'ram_size': 0x40000},
+              'npcx9m7': {'ram_address': 0x10070000, 'ram_size': 0x60000}}
```

Comment:
using a trailing command and putting the brace on the next line will make it easier to add another one (only need to add one line, instead of adding one line and modifying another.

Reply:
done.